src/Model/SearchHighlights.re

```diff
@@ -45,6 +45,7 @@ let highlightRangesToMap = (ranges: list(Range.t)) => {
 let reduce = (action: Actions.t, state: t) => {
   switch (action) {
   | SearchSetMatchingPair(bid, startPos, endPos) =>
+    Log.debug("Handling SearchSetMatchingPair");
```

Comment:
should these be removed again?

Reply:
were these here before? I've found them helpful so thought I would leave on debug